src/ng-backstretch.js

```diff
@@ -21,9 +21,9 @@ directive('backstretch', ['$window', '$timeout', function($window, $timeout) {
        * e.g. <div backstretch backstretch-url="'/path/to/image.jpg'">
        * So, we need to turn this back into an array.
        */
-      scope.images = Array.isArray(scope.images()) ? scope.images() : [scope.images()];
-      scope.duration = scope.duration() || 5000;
-      scope.fade = scope.fade() || 1;
+      scope.images = Array.isArray(scope.images) ? scope.images : [scope.images];
+      scope.duration = !angular.isUndefined(scope.duration) ? scope.duration : 5000;
```

Comment:
I'll take another look but, in the demo I created, I was able to change the duration and fade values from the attributes.